components/tag/stories/tag.stories.js

```diff
@@ -100,15 +103,21 @@ export default {
 		},
 		packageJson,
 		metadata,
+		downState: {
+			selectors: [".spectrum-Tag:not(:disabled)"],
+		},
 	},
+	decorators: [
+		withDownStateDimensionCapture,
```

Comment:
I've noticed this sometimes stops working if you switch to the testing grid and back, but this happens with other components too. Something else I'd like to follow up on. It generally works fine if you reload.

Reply:
There's a ticket for this!

Reply:
Yeah, usually it resolves itself and starts defining those custom props again if you reload, at least for me. I think that's mentioned in the ticket but if you're seeing otherwise, please add to it!